### DISCUSSION TITLE
Include multiple schemas into one prisma schema file

### DISCUSSION THREAD

**Discussion by andreasvh-conceto** [prisma/prisma#13387] [Q&A] [answered]
Hello,

currently i am facing the issue, that a prisma.schema file becomes really large. Is there any way to split them, merge them with some include statements or something like this?

Thx a lot and best regards
Andreas

**Comment by psikoi** [comment, marked as answer]
There's an issue for this: https://github.com/prisma/prisma/issues/2377

There's a few tools that do this I believe, one of them is [`prisma-aurora`](https://github.com/sabinadams/aurora) developed by [Sabin Adams](https://github.com/sabinadams) (works at Prisma).

He showcased it on [`What's new in Prisma (v3.12.0)`](https://youtu.be/TGMF8ykS29M?t=2757) at around the 46 minute mark.

**Reply by andreasvh-conceto** [reply]
Thanks a lot, that is something i was looking for :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
